### ISSUE TITLE
Why can't I use the styled-components over the <Terminal />?

### ISSUE THREAD

**Issue by Bruce-zxy** [nitin42/terminal-in-react#56]
I use the styled-components over the Terminal like this:
![image](https://user-images.githubusercontent.com/22904758/39675022-a0fd04ae-5187-11e8-88b9-75beb9a02bef.png)
![image](https://user-images.githubusercontent.com/22904758/39675029-c3e8caca-5187-11e8-9c12-b27817d0f655.png)

But the style that originally belonged to the Termianl is gone, it looks like this:
![image](https://user-images.githubusercontent.com/22904758/39675074-6a55a734-5188-11e8-8ca9-cbd62f75671f.png)

I didn't konw what was going on in the first place, thinking I didn't load the style of Terminal. But when I got rid of the styled-components, the Terminal was suddenly normal.
![image](https://user-images.githubusercontent.com/22904758/39675111-1311e356-5189-11e8-9129-6afeeecfeca1.png)
![image](https://user-images.githubusercontent.com/22904758/39675115-21dc173a-5189-11e8-938a-4ae12827a925.png)

So who can tell me why? And I found that they both had the same class name.
![image](https://user-images.githubusercontent.com/22904758/39675158-bda9b15e-5189-11e8-99d1-6dc46719b39e.png)

It made me very confused.
Help me, please.

**Comment by jcgertig** [comment]
Im not an expert on styled-components but we use styled-components so
likely you are overriding our styles with your style wapper.

**Comment by nitin42** [comment]
I think @jcgertig is right. You're overriding our styles (renderer with Styled-Components). Have you tested this behaviour with other css-in-js solutions ? Like emotion, glamorous ?

**Comment by nitin42** [comment]
If it works well with other css-in-js libs. then I guess the issue is with styled-components and not with our `Terminal` component!

**Comment by Bruce-zxy** [comment]
![image](https://user-images.githubusercontent.com/22904758/39677096-0acd23d0-51a8-11e8-8e0c-718a47c35ad5.png)

**Comment by Bruce-zxy** [comment]
When I import my own style through styled-components, the style of this class has disappeared, but it's not overriding.

![image](https://user-images.githubusercontent.com/22904758/39677118-6a012a90-51a8-11e8-877e-03006ffa1fcf.png)
![image](https://user-images.githubusercontent.com/22904758/39677136-abd07f98-51a8-11e8-9bd9-91c8f2a3d7e3.png)


And when I commented out the code, the style of this class name was appeared;
![image](https://user-images.githubusercontent.com/22904758/39677124-83daf09a-51a8-11e8-9231-7fe11daee313.png)
![image](https://user-images.githubusercontent.com/22904758/39677143-c5d66650-51a8-11e8-9b15-7943386e8291.png)

I don't means that the problem is caused by Terminal component, because I also consider it was caused by styled-components. But I'm sorry I can't find the cause of the problem.
